core/templates/dev/head/css/oppia.css

```diff
@@ -671,16 +671,47 @@ textarea {
   bottom: 0;
   color: #fff;
   font-size: 13px;
-  line-height: 20px;
-  position: absolute;
+  line-height: 10px;
```

Comment:
Got it, this change was a result of an earlier implementation. Have changed back.